### PR TITLE
Feature/issue 34/new delete organization flow

### PR DESCRIPTION
**Pull request by TekuriSaiAkhil**
# [Issue 34](https://github.com/gcivil-nyu-org/wed-fall24-team5/issues/34)

## Description

Modified the delete organization flow, which is
- Makes all the reservations inactive
- Makes all the donations inactive
- Makes the organization inactive

## Change Type (delete non-relevant options)

- 💡 New feature (non-breaking change which adds functionality)

## Dependencies

- None

### CODE REVIEW

**Review by maxgrove42 — APPROVED** [review]
nice implementation!

File: src/donor_dashboard/templates/donor_dashboard/list.html [hunk 1]
```diff
@@ -51,49 +51,137 @@ <h2 class="subtitle mt-4">
...
-                    </div>
                 {% endfor %}
             {% else %}
-                <p class="box">You are not a part of any organizations yet!!</p>
+                <p class="box">No In Active Organizations!!</p>
             {% endif %}
         </div>
+        </div>
     </div>
+    <script>
```
Comment (arsalananwar11) [inline src/donor_dashboard/templates/donor_dashboard/list.html:170]: Can you please shift this to a new javascript file under static?

**Review by arsalananwar11 — CHANGES_REQUESTED** [review]
Everything looks great, except for the in-file javascript. Thanks Akhil!

File: src/donor_dashboard/templates/donor_dashboard/list.html (hunk 1 above)
Comment (TekuriSaiAkhil) [reply src/donor_dashboard/templates/donor_dashboard/list.html]: Thanks @arsalananwar11!!
Updated accordingly

**Comment by TekuriSaiAkhil** [comment]
> nice implementation! minor nit, but please make sure to include some tests for the delete functionality to keep coverage up !

I have created a new PR for tests.

**Review by arsalananwar11 — APPROVED** [review]
Looks great, thanks for doing this in a short time!

**Pull request merged**